addons/event/static/src/views/event_slot_calendar/event_slot_calendar_model.js

```diff
@@ -22,8 +24,9 @@ export class EventSlotCalendarModel extends CalendarModel {
     normalizeRecord(rawRecord) {
         const normalizedRecord = super.normalizeRecord(rawRecord);
         const tz = rawRecord.date_tz || 'utc';
-        normalizedRecord.start = normalizedRecord.start.setZone(tz);
-        normalizedRecord.end = normalizedRecord.end.setZone(tz);
+        // Object conversion to make the datetimes naive.
+        normalizedRecord.start = DateTime.fromObject(normalizedRecord.start.setZone(tz).toObject());
+        normalizedRecord.end = DateTime.fromObject(normalizedRecord.end.setZone(tz).toObject());
```

Comment:
Could use `setZone('local', {keepLocalTime: true})` to be more explicit.

Reply:
Hello :slightly_smiling_face: @reth-odoo 
Yes if I understand correctly when you create slots from the calendar, it's using the local timezone to render and the datetimes are saved in UTC in DB.
What I'm trying to do here is that, if I create a slot from 8AM to 10AM on 11th of June, it will actually be directly in the event tz so that we have the exact same datetimes proposed in the event registration form and the front-end.
It's far easier for understanding or maintenance purpose and even more useful because of the amount of slots to display.

I don't really know if there's an easier way then what I did but it seems to work fine :slightly_smiling_face:

Reply:
Also I think `setZone('local', {keepLocalTime: true})` will convert to the local tz but I don't want that :smile: 
I want to convert to the event tz (what I did here with `setZone(tz)`) and than just remove the event tz from the zone param :slightly_smiling_face: